FeedSX/screens/UniversalFeed/index.tsx

```diff
@@ -57,9 +69,42 @@ const UniversalFeed = () => {
     return getFeedResponse;
   }
 
+  // this functions hanldes the post like functionality
+  async function postLikeHandler(id: string) {
+    let payload = {
+      postId: id,
+    };
+    dispatch(likePostStateHandler(payload.postId) as any);
+    // calling like post api
+    let postLikeResponse = await dispatch(
+      likePost(
+        LikePostRequest.builder().setpostId(payload.postId).build(),
```

Comment:
it should `setPostId()`

Reply:
its automatically suggesting setpostId only